pyproject.toml

```diff
@@ -38,6 +38,8 @@ fastapi = "^0.105.0"
 numpy = "^1.26.3"
 pandas = "^2.2.0"
 pandas-stubs = "^2.1.4.231227"
+tiktoken = "^0.5.2"
```

Comment:
why are these added to our pyproject, instead of the requirements.txt

Reply:
it's added to both. weirdly github smushed this, if you expand you'll see this is added to the cookbook dependency group (I almost wrote the same comment)